WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -315,13 +325,23 @@ private async Task<(ImmutableArray<AliceClient> aliceClientsThatSigned, IEnumera
 		try
 		{
 			var roundId = roundState.Id;
+			var roundParameters = roundState.CoinjoinState.Parameters;
 
 			registeredAliceClientAndCircuits = await ProceedWithInputRegAndConfirmAsync(smartCoins, roundState, cancellationToken).ConfigureAwait(false);
 			if (!registeredAliceClientAndCircuits.Any())
 			{
 				throw new CoinJoinClientException(CoinjoinError.NoCoinsEligibleToMix, $"The coordinator rejected all {smartCoins.Count()} inputs.");
 			}
 
+			var smallestEffectiveDenom = roundParameters.CalculateSmallestReasonableEffectiveDenomination();
```

Comment:
This happens after coins were confirmed. We have no numbers so, we really don't know how often this will throw the exception but some rounds (very few i guess) will be disrupted by these lines.

Btw, here again, given `CalculateSmallestReasonableEffectiveDenomination` is not deterministic, here we can get a different result than before. The difference is so small than we could ignore it(?) but it doesn't look okay.